src/components/content/heroes/aurora/precipitationEffect.tsx

```diff
@@ -12,6 +12,18 @@ const fallAnimation = keyframes`
   }
 `;
 
+const swayAnimation = keyframes`
+  0% {
+    transform: translateX(0);
+  }
+  50% {
+    transform: translateX(${Math.random() * 400 - 200}px);
```

Comment:
Hmm, Math.random() only runs once here for all snowflakes, that is not what we intended. Please have a look how to randomize this for each snowflake.

Maybe this helps: https://medium.com/nerd-for-tech/may-14-random-css-animations-using-styled-components-in-react-dda83d9afb62